woocommerce/woocommerce-ios
# [CPP Onboarding] Implement view for plugin version not supported

Part of #4611 

This adds a specific onboarding error screen for the case when the version of WooCommerce Payments installed is lower than the minimum supported version.

I made up the error messages completely.

Also, this PR removes a warning in SettingsViewController. I am not sure if this is really necessary, but I noticed it so 🤷🏼 

## To test

There is a quick way to test it and a long and painful way to test this. Your choice! 

The quick way:
1. Edit CardPresentPaymentsOnboardingUseCase. In line 201, change:
`static let supportedWCPayVersion = "2.5"` 
to 
`static let supportedWCPayVersion = "5.0"`

2. Login to an store with WCPay installed (ideally, the store would be fully configured and gated into In-Person Payments
3. Notice the error message:

<img src="https://user-images.githubusercontent.com/2722505/128712956-b4156d8a-1558-40c8-8add-9c8ef22463bf.png" width="350"/>


The long and painful way:
1. Install a version of WCPay lower than 2.5
1. Go to Settings > In-Person Payments. The app should show a screen prompting you to update the extension.
2. From wp-admin, update WooCommerce Payments from WooCommerce > Extensions > Payments.
3. Tap on Refresh After Updating.
4. The app should show a generic error (since payments aren't fully set up yet).

Update release notes:
